tests/swagger_generation/registries/multiple_authenticators.py

```diff
@@ -220,7 +220,11 @@ class NameAndOtherSchema(m.Schema):
     headers_schema=HeaderSchema(),
 )
 def get_foo(foo_uid):
-    """helpful description"""
+    """
+    a summary
+
+    helpful description
+    """
     pass
```

Comment:
We should make sure there is a test for the description only case, as well as the no docstring case.